### PR TITLE
Update dialoguekit imports

### LINKED ISSUE #72: Bug DialogueKit imports
The following command throw ModuleNotFoundError with DialogueKit 0.0.5

```
python -m usersimcrs.run_simulation -ontology data/movies.yaml
> Traceback (most recent call last):
  File "/Users/2925364/miniconda3/envs/usersimcrs_test/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/Users/2925364/miniconda3/envs/usersimcrs_test/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/2925364/Documents/UserSimCRS/usersimcrs/run_simulation.py", line 17, in <module>
    from dialoguekit.platformss.platform import Platform
ModuleNotFoundError: No module named 'dialoguekit.platformss'
```

After fixing the typo (i.e., 'dialoguekit.platforms' instead of 'dialoguekit.platformss'), another ModuleNotFoundError is returned.
```
python -m usersimcrs.run_simulation -ontology data/movies.yaml
> Traceback (most recent call last):
  File "/Users/2925364/miniconda3/envs/usersimcrs_test/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/Users/2925364/miniconda3/envs/usersimcrs_test/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/2925364/Documents/UserSimCRS/usersimcrs/run_simulation.py", line 20, in <module>
    from usersimcrs.simulator.agenda_based.agenda_based_simulator import (
  File "/Users/2925364/Documents/UserSimCRS/usersimcrs/simulator/agenda_based/agenda_based_simulator.py", line 10, in <module>
    from usersimcrs.simulator.preference_model import PreferenceModel
ModuleNotFoundError: No module named 'usersimcrs.simulator.preference_model'
```

I upgraded DialogueKit to 0.0.6 but I still get ModuleNotFoundError.
```
python -m usersimcrs.run_simulation  -ontology data/movies.yaml
> Traceback (most recent call last):
  File "/Users/2925364/miniconda3/envs/usersimcrs_test/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/Users/2925364/miniconda3/envs/usersimcrs_test/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/2925364/Documents/UserSimCRS/usersimcrs/run_simulation.py", line 9, in <module>
    from dialoguekit.agent.agent import Agent
  File "/Users/2925364/miniconda3/envs/usersimcrs_test/lib/python3.8/site-packages/dialoguekit/agent/__init__.py", line 3, in <module>
    from dialoguekit.agent.mathematics_agent import MathAgent
  File "/Users/2925364/miniconda3/envs/usersimcrs_test/lib/python3.8/site-packages/dialoguekit/agent/mathematics_agent.py", line 13, in <module>
    from dialoguekit.nlg.nlg import NLG
ModuleNotFoundError: No module named 'dialoguekit.nlg.nlg'
```

### PR DESCRIPTION
**Pull request by NoB0**
- Fix dialoguekit imports
- Rename ontology by domain

Fixes #72

### CODE REVIEW

File: tests/simulator/user_modeling/test_preference_model.py [hunk 1]
```diff
@@ -55,59 +56,45 @@ def test_initial_item_preferences_sip(preference_model_sip):
...
-    # Then
     assert preference == 1
 
 
 def test_get_slot_preference_pkg(preference_model_pkg):
-    # Given
     slot = "GENRE"
 
-    # When
     slot, preference = preference_model_pkg.get_slot_preference(slot)
```
Comment (kbalog) [inline tests/simulator/user_modeling/test_preference_model.py:96]: just put the slot value inline and remove the line above

File: tests/simulator/user_modeling/test_preference_model.py [hunk 2]
```diff
@@ -55,59 +56,45 @@ def test_initial_item_preferences_sip(preference_model_sip):
 
 
 def test_get_slotvalue_preference_sip(preference_model_sip):
-    # Given
     slot = "GENRE"
     value = "Comedy"
 
-    # When
     preference = preference_model_sip.get_slotvalue_preference(slot, value)
```
Comment (kbalog) [inline tests/simulator/user_modeling/test_preference_model.py:62]: Put the actual values inline

File: tests/simulator/user_modeling/test_preference_model.py [hunk 3]
```diff
@@ -55,59 +56,45 @@ def test_initial_item_preferences_sip(preference_model_sip):
...
     assert -1 <= preference <= 1
 
 
 def test_get_slotvalue_preference_pkg(preference_model_pkg):
-    # Given
     slot = "GENRE"
     value = "Drama"
 
-    # When
     preference = preference_model_pkg.get_slotvalue_preference(slot, value)
```
Comment (kbalog) [inline tests/simulator/user_modeling/test_preference_model.py:71]: Put the actual values inline

File: usersimcrs/simulator/user_simulator.py [hunk 4]
```diff
@@ -2,15 +2,15 @@
 
 from abc import ABC, abstractmethod
 
-from dialoguekit.user.user import User, UserType
 from dialoguekit.core.utterance import Utterance
+from dialoguekit.particpant.user import User, UserType
```
Comment (kbalog) [inline usersimcrs/simulator/user_simulator.py:6]: typo in package name

**Review by kbalog — CHANGES_REQUESTED** [review]
Many thanks for proactively cleaning up the tests. I really didn't like how the # Given, #When, # Then structuring was done. Since you started on it already, would you mind just putting in all the values inline (i.e., directly as arguments) as opposed to declaring them as variables? (I put in comments for some of these instances, but not all...)

File: usersimcrs/simulator/user_simulator.py (hunk 4 above)
Comment (NoB0) [reply usersimcrs/simulator/user_simulator.py]: Done.

File: tests/simulator/user_modeling/test_preference_model.py (hunk 3 above)
Comment (NoB0) [reply tests/simulator/user_modeling/test_preference_model.py]: Done.

File: tests/simulator/user_modeling/test_preference_model.py (hunk 2 above)
Comment (NoB0) [reply tests/simulator/user_modeling/test_preference_model.py]: Done.

File: tests/simulator/user_modeling/test_preference_model.py (hunk 1 above)
Comment (NoB0) [reply tests/simulator/user_modeling/test_preference_model.py]: Done.

**Review by kbalog — APPROVED** [review]
LGTM with a minor comment

**Pull request merged**